### PR TITLE
Send data

### PR DESCRIPTION
**Pull request by pashalee1105**

### CODE REVIEW

File: btutillib/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.BLUETOOTH"/>
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```
Comment (ptsouchlos) [inline btutillib/src/main/AndroidManifest.xml:7]: Can you comment as to why this permission was added?

**Review by ptsouchlos — COMMENTED** [review]
@pashalee1105  Was this pull request accidental? Did you mean to delete that branch?

**Comment by pashalee1105** [comment]
@DeveloperPaul123  Yes, It's accidental and I want to delete this branch. Sorry for any inconvenience it may cause.

**Comment by ptsouchlos** [comment]
@pashalee1105 No problem. Feel free to re-open it though. I was planning on merging the changes.

**Pull request closed without merging**